dataactbroker/handlers/accountHandler.py

```diff
@@ -173,12 +174,12 @@ def ThreadedFunction (from_email="",username="",title="",agency="",userEmail=""
             threadedDatabase =  UserHandler()
             try:
                 for user in threadedDatabase.getUsersByType("website_admin"):
-                    emailTemplate = {'[REG_NAME]': username, '[REG_TITLE]':title, '[REG_AGENCY]':agency,'[REG_EMAIL]' : userEmail,'[URL]':link}
+                    emailTemplate = {'[REG_NAME]': username, '[REG_TITLE]':title, '[REG_AGENCY]':cgac_code,'[REG_EMAIL]' : userEmail,'[URL]':link}
```

Comment:
Does it make sense to use the CGAC description instead of the code in these e-mail templates (or some combination of both?). I suspect the CGAC codes aren't meaningful for most people.